protocols/enocean/com.orange.enocean.eeps/bnd.bnd

```diff
@@ -0,0 +1,27 @@
+# Set javac settings from JDT prefs
+-include: ${build}/eclipse/jdt.bnd
+
+Bundle-Activator  : com.orange.enocean.eeps.Activator
+Bundle-Copyright  : Copyright (C) 2013 - 2015 Orange.
+Bundle-Description: OSGi EnOcean bundle that contained the three EEPs that the EnOcean Alliance allowed the project to use.
+Bundle-License    : Apache License, Version 2.0
+Bundle-RequiredExecutionEnvironment: OSGi/Minimum-1.1
+Bundle-Vendor     : Orange
+Bundle-Version    : 1.0.0
```

Comment:
Do you have any requirements regarding the bundle version? If not, it would make sense to synchronize this with the Eclipse SmartHome version, which is 0.8.0.